packages/react-button/src/components/ToggleButton/useToggleButtonStyles.ts

```diff
@@ -156,227 +149,38 @@ export const makeToggleButtonTokens = (theme: Theme): ToggleButtonVariantTokens
-  },
-  rootDisabled: theme => {
-    const toggleButtonTokens = makeToggleButtonTokens(theme);
-
-    return {
-      background: toggleButtonTokens.disabled?.background,
-      borderColor: toggleButtonTokens.disabled?.borderColor,
-      color: toggleButtonTokens.disabled?.color,
-
-      boxShadow: toggleButtonTokens.disabled?.shadow,
-
-      ':hover': {
-        background: toggleButtonTokens.disabled?.hovered?.background,
-        borderColor: toggleButtonTokens.disabled?.hovered?.borderColor,
-        color: toggleButtonTokens.disabled?.hovered?.color,
-
-        boxShadow: toggleButtonTokens.disabled?.hovered?.shadow,
-      },
-
-      ':active': {
-        background: toggleButtonTokens.disabled?.pressed?.background,
-        borderColor: toggleButtonTokens.disabled?.pressed?.borderColor,
-        color: toggleButtonTokens.disabled?.pressed?.color,
-
-        boxShadow: toggleButtonTokens.disabled?.pressed?.shadow,
-      },
-    };
-  },
-  rootDisabledPrimary: theme => {
-    const buttonTokens = makeToggleButtonTokens(theme);
-
-    return {
-      borderColor: buttonTokens.disabledPrimary?.borderColor,
-
-      ':hover': {
-        borderColor: buttonTokens.disabledPrimary?.hovered?.borderColor,
-      },
-
-      ':active': {
-        borderColor: buttonTokens.disabledPrimary?.pressed?.borderColor,
-      },
-    };
-  },
-  rootDisabledSubtle: theme => {
-    const buttonTokens = makeToggleButtonTokens(theme);
-
-    return {
-      background: buttonTokens.disabledSubtle?.background,
-      borderColor: buttonTokens.disabledSubtle?.borderColor,
-
-      ':hover': {
-        background: buttonTokens.disabledSubtle?.hovered?.background,
-        borderColor: buttonTokens.disabledSubtle?.hovered?.borderColor,
-      },
-
-      ':active': {
-        background: buttonTokens.disabledSubtle?.pressed?.background,
-        borderColor: buttonTokens.disabledSubtle?.pressed?.borderColor,
-      },
-    };
-  },
-  rootDisabledTransparent: theme => {
-    const buttonTokens = makeToggleButtonTokens(theme);
-
-    return {
-      background: buttonTokens.disabledTransparent?.background,
-      borderColor: buttonTokens.disabledTransparent?.borderColor,
-
-      ':hover': {
-        background: buttonTokens.disabledTransparent?.hovered?.background,
-        borderColor: buttonTokens.disabledTransparent?.hovered?.borderColor,
-      },
-
-      ':active': {
-        background: buttonTokens.disabledTransparent?.pressed?.background,
-        borderColor: buttonTokens.disabledTransparent?.pressed?.borderColor,
-      },
-    };
-  },
-});
-
-export const useToggleButtonStyles = (state: ToggleButtonState, selectors: ToggleButtonStyleSelectors) => {
+export const useToggleButtonStyles = (state: ToggleButtonState): ToggleButtonState => {
   // Save the classnames used in useButtonStyles and undefine them at the state level so that they are always applied
   // last.
   const { className: rootClassName } = state;
   useButtonStyles(state);
```

Comment:
Same comment as in CompoundButton, can you move `useButtonStyles` to the end instead of doing this? (Or if not: the comment says that you're undefining the className, but you don't actually do that here.)

Reply:
For sure, will make the change, thanks for the review!